config/routes.rb

```diff
@@ -1,4 +1,22 @@
 Rails.application.routes.draw do
+  namespace :tpi do
+    resources :sectors, only: [:show, :index] do
```

Comment:
should we have `API` namespace also? We can add that later too.

Reply:
Not sure what we would want to put in `api` namespace. Do you mean API for 2nd laws frontend?